basics.js

```diff
@@ -41,16 +58,17 @@ var anna = {name: "Anna",
             }
           }
 
-// What is `anna`? (e.g. method, function, string, etc.) 
-
+// What is `anna`? (e.g. method, function, string, etc.) - Object literal
 
-// What are `name`, `age`, and `luckyNumber`? 
+// What are `name`, `age`, and `luckyNumber`? - properties
 
 
-// What does `this` refer to? 
+// What does `this` refer to? - The object literal, in this case, anna.
 
 
-// If you want to return the value of anna's lucky number, how would you do that? 
+// If you want to return the value of anna's lucky number, how would you do that?
+return anna.luckyNumber;
```

Comment:
There's no need to return from the global level, but I understand your logic. But more importantly, you need () at the end of the function call to invoke it. `anna.luckyNumber()`